src/components/buttons/Button.jsx

```diff
@@ -66,10 +66,12 @@ type ButtonIconType =
   | {
       icon?: Node,
       iconOnly?: null,
+      reversedOrder?: boolean,
     }
   | {
       icon: Node,
       iconOnly?: boolean,
+      reversedOrder?: null,
```

Comment:
If there's no icon, flow will complain when we pass reversedOrder. But technically, we can add `icon`, `iconOnly` and `reversedOrder` (which doesn't make sense), and flow will not complain. I would leave this as is, because fixing that would introduce more unnecessary complexity imho (we would have to add more interfaces here). Wdyt?

Reply:
i think it's acceptable for now till we find a solution how to make it simpler